test/extended/networking/egress_firewall.go

```diff
@@ -128,6 +149,21 @@ func sendEgressFwTraffic(f *e2e.Framework, oc *exutil.CLI, pod string, nodeSelec
 	_, err = oc.Run("exec").Args(pod, "--", "curl", "-q", "-s", "-I", "-m3", "https://docs.openshift.com").Output()
 	expectNoError(err)
 
+	// TODO:
+	// - Check for coredns-ocp-dnsnameresolver plugin.
+	// - Remove TechPreview check when feature is GA.
+	if networkPluginName() == OVNKubernetesPluginName && exutil.IsTechPreviewNoUpgrade(oc) {
```

Comment:
I think this should be just an argument of sendEgressFwTraffic, like `checkWildcard`, and it can be propagated to `doEgressFwTest`. Then we won't need extra plugin + feature gate check

Reply:
I will make the change to pass it as an arg.
Since this was for TP, I thought not to change the function signature.